[ODS-6152] Migrate DataLoading solution to .NET 8

axelmarquezh opened the pull request:
I've also updated the related libraries in our artifacts repository:
- EdFi.Suite3.LoadTools
- EdFi.Suite3.BulkLoadClient.Console

simpat-jesus reviewed (changes requested):
Some dependencies also need to be upgraded. Please use the [assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant) to handle those dependencies.

axelmarquezh commented:
Thanks Jesus. I've updated the dependencies as well.
Note that I also removed the `EdFi.XmlLookup.Console` project, which seems to have been missed while developing ODS-6084.
